frontend/include/chpl/framework/ErrorMessage.h

```diff
@@ -20,38 +20,26 @@
 #ifndef CHPL_QUERIES_ERRORMESSAGE_H
 #define CHPL_QUERIES_ERRORMESSAGE_H
 
-#include "chpl/framework/Location.h"
-#include "chpl/framework/ID.h"
-#include "chpl/util/printf.h"
-
 #include <cstdarg>
 #include <string>
 #include <utility>
 #include <vector>
 
-namespace chpl {
-
-// forward declare AstNode
-namespace uast {
-  class AstNode;
-}
+#include "chpl/framework/ID.h"
+#include "chpl/framework/Location.h"
+#include "chpl/util/printf.h"
 
+namespace chpl {
 
 /**
   This class represents an error/warning message. The message
   is saved (in the event it needs to be reported again).
  */
 class ErrorMessage final {
  public:
-  enum Kind {
-    NOTE,
-    WARNING,
-    SYNTAX,
-    ERROR
-  };
+  enum Kind { NOTE, WARNING, SYNTAX, ERROR };
```

Comment:
Just making sure this is deliberate.

Reply:
The auto-formatter's fault. I don't have a strong preference between the two, but I'll put it back to the one-per-line.